src/app/auth/auth.service.ts

```diff
@@ -21,6 +28,15 @@ export class AuthService {
 
     return this.request.post('api/auth.json', body.toString(), { 
     	headers: { 'Content-Type': 'application/x-www-form-urlencoded' }
-    });
+    }).pipe(map(response => {
+      console.log('Auth Response::', response);
+      this.storage.set('apikey', response.data.apikey);
+      return response;
+    }));
   }
+
+  saveRegistration(data): Observable<any> {
+    return of(true);
```

Comment:
use this as the mock API return, and component should respond based on "success" 
```
{
  "success": true
}
```